services/app_api/swagger.yaml

```diff
@@ -0,0 +1,334 @@
+openapi: 3.0.0
```

Comment:
Is this needed?

Reply:
This is my last todo.  It is up to the code challenge peeps whether or not they want to implement swagger in the boiler plate